### PR TITLE
add "Trending"

### LINKED ISSUE #153: Show "Trending" in the top menu
From https://github.com/isaacs/github/issues/245.

Currently the link to your starred repos is hidden in the avatar dropdown. Might be nice to add it to the top menu. Not entirely sure about this, so use "Reactions" to vote whether people want this or not.

napkin 04-04-16 3 47 15 pm

### PR DESCRIPTION
**Pull request by jacobbearden**
closes #153

Not sure if this is the optimal way to do it, let me know if there's a better way.

### CODE REVIEW

File: extension/content.js [hunk 1]
```diff
@@ -68,6 +68,15 @@ function addReleasesTab() {
 	}
 }
 
+function addTrendingMenuItem() {
+	const $secondButton = $('.header-nav.left .header-nav-item:nth-child(2)');
+	const $trendingButton = $(`<li class="header-nav-item">
```
Comment (paulmolluzzo) [inline extension/content.js]: This is a nitpick, but shouldn't it be `$secondListItem` and `$trendingListItem`? They're definitely not `button` elements.

**Comment by paulmolluzzo** [comment]
@jacobbearden thanks for this one too! Already nice to have this link in the header. 😄

File: extension/content.js (hunk 1 above)
Comment (jacobbearden) [inline extension/content.js]: @paulmolluzzo fixed!

**Comment by sindresorhus** [comment]
Thanks :)

**Pull request closed without merging**